### PR TITLE
Create nardcrat

### PR DESCRIPTION
**Pull request by meetkool**
To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: 
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

**Comment by meetkool** [comment]
using this for the protfolio site

File: nardcrat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "use for the main site for protfolio",
+    "repo": "https://github.com/meetkool/meetkool",
+    "owner": {
+        "username": "meetkool",
+        "email": "doomlol@protonmail.com"
+    },
+    "record": {
+        "CNAME": "meetssh.ml"
```
Comment (WilliamDavidHarrison) [inline nardcrat:9]: Please add more content to your website.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please rename your file to `nardcrat.json`

File: nardcrat (hunk 1 above)
Comment (meetkool) [reply nardcrat:9]: right now it is for testing i will add

**Comment by meetkool** [comment]
ya just changed to nardcrat.json

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please delete the file without .json

**Comment by meetkool** [comment]
@WilliamDavidHarrison done i deleted

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add more content to your website.

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**